lang/fpc/Portfile

```diff
@@ -3,94 +3,169 @@
 PortSystem          1.0
 
 name                fpc
-version             2.6.4
-revision            1
+version             3.0.4
 categories          lang
 platforms           darwin
 license             GPL-2 LGPL-2
 maintainers         nomaintainer
-description         free pascal
-long_description    Open source compiler for Pascal and Object Pascal
+description         Free Pascal, an open source Pascal and Object Pascal compiler.
+long_description    Free Pascal is a 32, 64 and 16 bit professional Pascal compiler. \
+                    It can target many processor architectures: Intel x86 (including 8086), \
+                    AMD64/x86-64, PowerPC, PowerPC64, SPARC, ARM, AArch64, MIPS and the JVM. \
+                    Supported operating systems include Linux, FreeBSD, Mac OS X/iOS/iPhoneSimulator/Darwin, \
+                    Win32, Win64, WinCE and Android.
 homepage            http://www.freepascal.org
 master_sites        sourceforge:freepascal
 
 set src             ${name}build-${version}.tar.gz
-set pp              universal-macosx-10.5-ppcuniversal.tar.bz2
+# the 3.0.0 bootstrap compiler
+set pp              x86_64-macosx-10.7-ppcx64.tar.bz2
```

Comment:
It would be better to not hardcode the suffix to keep these generic, so change these to:

```
set src             ${name}build-${version}${extract.suffix}
set pp              x86_64-macosx-10.7-ppcx64${extract.suffix}
```

Reply:
Ignore this comment. I just realized that the suffixes are different.